pep/app/pods/components/page/sidebar/widgets/favorites/component.ts

```diff
@@ -25,8 +32,20 @@ export default class PageSidebarWidgetsFavorites extends Component<PageSidebarWi
      *
      * @memberof PageSidebarWidgetsFavorites
      */
-    loadFromUserPreferences() {
-        this.results = this.currentUser.getDocuments(PreferenceKey.FAVORITES);
+    @restartableTask
+    *loadFromUserPreferences() {
+        const ids = this.currentUser.getDocuments(PreferenceKey.FAVORITES);
```

Comment:
could we put a `TODO` comment here about adding the smart pulling of documents from the local cache?